src/App.tsx

```diff
@@ -1,24 +1,216 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { UserWarning } from './UserWarning';
+import {
+  getTodos,
+  addTodo,
+  deleteTodo,
+  completedTodo,
+  updateTodoTitle,
+} from './api/todos';
+import { Todo } from './types/Todo';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+import { FilterBy } from './types/FilterBy';
+import { Notification } from './components/Notification';
 
-const USER_ID = 0;
+const USER_ID = 6259;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState<FilterBy>(FilterBy.All);
+  const [isError, setIsError] = useState(false);
+  const [errorText, setErrorText] = useState('');
+  const [todoTitle, setTodoTitle] = useState('');
+  const [disableInput, setDisableInput] = useState(false);
+  const [editingTodosId, setEditingTodosId] = useState<number[]>([]);
+  const [formShowedForId, setFormShowedForId] = useState(0);
+
+  const getAllTodos = async () => {
+    try {
+      setDisableInput(true);
+      const data = await getTodos(USER_ID);
+
+      setTodos(data);
+      setDisableInput(false);
+    } catch (error) {
+      setErrorText('Error with loading todos');
+      setIsError(true);
+      throw new Error('Error with loading todos');
+    }
+  };
+
+  useEffect(() => {
+    getAllTodos();
+  }, []);
+
+  const handleAddTodo = async (todoText: string) => {
+    if (todoTitle.length === 0) {
```

Comment:
```suggestion
    if (!todoTitle.trim()) {
```
empty array or only with spaces should not appear